commands/cli/parse.go

```diff
@@ -357,7 +358,13 @@ func appendStdinAsString(args []string, stdin *os.File) ([]string, *os.File, err
 }
 
 func appendFile(args []files.File, inputs []string, argDef *cmds.Argument, recursive bool) ([]files.File, []string, error) {
-	fpath := inputs[0]
+	fpath := filepath.Clean(inputs[0])
+	if fpath[:2] == "\\\\" {
```

Comment:
Also, https://golang.org/src/path/filepath/path_windows.go#L40-L63 implements the UNC check.